corehq/apps/userreports/es/data_source.py

```diff
@@ -93,12 +95,39 @@ def _get_query(self, start=None, limit=None):
 
     @memoized
     def _get_aggregated_results(self, start, limit):
+        def _parse_bucket(bucket, remaining_agg_columns, current_bucket_name, past_bucket_info):
```

Comment:
Originally looked into `NestedTermsHelper` to fix this one, but it doesn't quite do what I want it to. I'm looking into fixing that, but thought it would be good get this out to get the error out of the way